components/record/MetadataField.vue

```diff
@@ -22,6 +24,18 @@
       value: {
         type: [String, Object],
         default: ''
+      },
+      context: {
+        type: String,
+        default: 'default',
+        validator: (value) => {
+          return fieldLabellingContexts().indexOf(value) !== -1;
+        }
+      }
+    },
+    computed: {
+      fieldNameLabel: function() {
+        return fieldLabel(this.name, { context: this.context });
```

Comment:
It is not a reserved word, but you'd better avoid "name".
See also: https://www.w3schools.com/js/js_reserved.asp

Reply:
As discussed let's leave it for now, if it's causing us issues down the line we may refacotr with something like 'fieldName'.